sdk/src/auth/composable.ts

```diff
@@ -18,19 +19,12 @@ const defaultConfigValues = {
  *
  * @returns A Directus authentication client.
  */
-export const authentication = (mode: AuthenticationMode = 'cookie', config: AuthenticationConfig = {}) => {
+export const authentication = (mode: AuthenticationMode = 'cookie', config: Partial<AuthenticationConfig> = {}) => {
 	return <Schema extends object>(client: DirectusClient<Schema>): AuthenticationClient<Schema> => {
-		config = { ...defaultConfigValues, ...config };
+		const authConfig = { ...defaultConfigValues, ...config };
 		let refreshPromise: Promise<AuthenticationData> | null = null;
 		let refreshTimeout: NodeJS.Timer | null = null;
-		const storage = config.storage ?? memoryStorage();
-
-		const autoRefresh = 'autoRefresh' in config ? config.autoRefresh : defaultConfigValues.autoRefresh;
-
-		const msRefreshBeforeExpires =
-			typeof config.msRefreshBeforeExpires === 'number'
-				? config.msRefreshBeforeExpires
-				: defaultConfigValues.msRefreshBeforeExpires;
+		const storage = authConfig.storage ?? memoryStorage();
```

Comment:
Couldn't this be in `defaultConfigValues` as well?

Reply:
how do you mean? the `defaultConfigValues` are used to build the `authConfig` (similarly to `restConfig` and `gqlConfig`)

Reply:
Oh, right 😺 Little oversight from my side!